src/renderer/pages/player-page.js

```diff
@@ -216,9 +216,7 @@ function renderAudioMetadata (state) {
   const elems = []
 
   // Audio metadata: artist(s)
-  const artist = common.albumartist || common.artist ||
-    (common.artists && common.artists.filter(function (a) { return a }).join(', ')) ||
```

Comment:
`common.artists` array is not available anymore?

Reply:
It is still available, but common.artist is already designed to summarize the artist(s) in a single string.
Either derived from common.artists or directly from the metadata.

In other words, the code removed is already done in music-metadata.